external/header.php

```diff
@@ -34,16 +34,17 @@ public static function __callstatic($name, $arguments)
 // Only users from authorized IP addresses may control Profiling
 if ($controlIPs === false || in_array($_SERVER['REMOTE_ADDR'], $controlIPs) || PHP_SAPI == 'cli')
 {
-  if (isset($_GET['_profile']))
+  if (isset($_GET[$_xhprof['getparam']]))
```

Comment:
Why this needs to be configurable at all? There are no ties to PHP extension, so it can stay `_profile` to preserve BC.